tests/integration_test.rs

```diff
@@ -214,7 +220,7 @@ async fn test_policy_evaluator(
 )]
 #[case::wapc(
     PolicyExecutionMode::KubewardenWapc,
-    "ghcr.io/kubewarden/tests/context-aware-test-policy:v0.1.0",
+    "ghcr.io/jvanz/policies/context-aware-test-policy:latest",
```

Comment:
see comment: https://github.com/kubewarden/policy-evaluator/pull/429/commits/ee4a2d7d857472024a4b40cac2f4ce069cb28456#r1479439169

Reply:
This change is wrong. It's left over of other test. I'll remove it